components/AdForm/AdPositions/Position.js

```diff
@@ -41,32 +34,50 @@ class Position extends Component {
   setType = value => this.setState({ type: value });
   setCustomPostTypes = value => this.setState({ customPostTypes: value });
 
+  renderItemSelector = () => {
+    const { member } = this.props;
+    const { type, customPostTypes } = this.state;
+    const { items } = types[type];
```

Comment:
I'm getting an error `Cannot read property 'items' of undefined` here when visiting some of the sites.

The problem look related to adding `customPostTypes` to the items. I've managed to reproduce it going to a site and creating a new ad with a new "custom" position. After clicking on "Submit" that add disappears and then, the next time you visit that url, it fails with the `Cannot read property 'items' of undefined` error.

I've debugged this line and the `type` passed to `types` is "custom", but in the file types.js there's not a "custom" type but a "customPostType".

Changing that to "custom" in the types.js file solves the problem:
https://github.com/frontity/ads-form/blob/15-type-media/components/AdForm/types.js#L24
but obviously the problem comes from somewhere else, where it's saving "custom" in the database instead of "customPostType".

Reply:
It looks like I've found the place where it should be changed:
https://github.com/frontity/ads-form/blob/dev/components/AdForm/formats.js#L12

I'll push a commit.

Reply:
Ok, fixed.